gammapy/maps/wcsnd.py

```diff
@@ -518,22 +519,19 @@ def smooth(self, radius, kernel="gauss", **kwargs):
         """
         from scipy.ndimage import gaussian_filter, uniform_filter, convolve
 
-        if isinstance(radius, u.Quantity):
-            radius = (radius.to("deg") / self.geom.pixel_scales.mean()).value
+        if isinstance(width, u.Quantity):
+            width = (width.to("deg") / self.geom.pixel_scales.mean()).value
 
         smoothed_data = np.empty_like(self.data)
 
         for img, idx in self.iter_by_image():
             if kernel == "gauss":
-                width = radius / 2.
-                data = gaussian_filter(img, width, **kwargs)
+                data = gaussian_filter(img, width/2.0, **kwargs)
```

Comment:
I guess this should be rather `gaussian_filter(img, width, **kwargs)`. The [gaussian_filter](https://docs.scipy.org/doc/scipy-0.16.1/reference/generated/scipy.ndimage.filters.gaussian_filter.html) method takes  `sigma` as an argument. With this change you probably have to adapt the test as well...